src/FSharp.Examples/FSharp.Examples.fsproj

```diff
@@ -21,7 +24,8 @@
 
   <ItemGroup>
     <PackageReference Include="SharpZipLib" Version="1.3.3" />
-    <PackageReference Include="System.Memory" Version="4.5.3" />
+    <PackageReference Include="System.Memory" Version="4.5.4" />
+    <PackageReference Include="System.ValueTuple" Version="4.5.0" />
```

Comment:
NIT: I suppose this package reference is only needed in net472, if so you can condition it.